apps/monitor-client/lib/features/HomePage.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { useState } from 'react';
+import { Route, Routes } from 'react-router-dom';
+import PropTypes from 'prop-types';
+
+import { Layout } from '../components/layout/Layout';
+import { Sidebar } from '../components/layout/Sidebar';
+import { Header } from '../components/layout/Header';
+import { Content } from '../components/layout/Content';
+
+import { ServerView } from './ServerView';
+import { UsersView } from './UsersView';
+import { SessionsView } from './SessionsView';
+
+export function HomePage({}) {
```

Comment:
lib/features/homepage/components

also, add to all folders index.js with export components within this folder

Reply:
I think making structure like that is complicated, I don't expect a lot of component inside particular feature